Sources/Sentry/SentryCrashIntegration.m

```diff
@@ -233,6 +233,8 @@ + (void)enrichScope:(SentryScope *)scope crashWrapper:(SentryCrashWrapper *)cras
 
 #if TARGET_OS_SIMULATOR
     [deviceData setValue:@(YES) forKey:@"simulator"];
+#else
+    [deviceData setValue:@(NO) forKey:@"simulator"];
```

Comment:
Makes sense thanks 🙏 . Please add some tests for this functionality.

Reply:
I don't think a test would make a lot of sense? We'd always be checking for a NO value?

Reply:
Hmm, no test is OK as well. We would have to add another level of abstraction, but that's overkill. Let's keep it as it is.